jest/setup.js

```diff
@@ -52,7 +55,9 @@ jest
         // Ensure this doesn't throw.
         try {
           Object.keys(dataBlob).forEach(key => {
-            this.items += dataBlob[key] && dataBlob[key].length;
+            const isImmutableMap = Map.isMap(dataBlob[key]);
+            this.items += isImmutableMap ?
+              (dataBlob[key] && !dataBlob[key].isEmpty()) : (dataBlob[key] && dataBlob[key].length);
```

Comment:
In this case `length` is a property of `dataBlob[key]`. It's adding the value of `length` to the total value of `this.items`. So if `length` is 3, we add that to the total. I think to make your code work you can replace `(dataBlob[key] && !dataBlob[key].isEmpty())` with simply `dataBlob[key].size` since you already checked that it's an immutable Map.